README.md

```diff
@@ -68,16 +111,217 @@ you can also chain components names `react-component <ComponentName> <ComponentN
         |--- SideBarItem.js
         |
         |--- SideBarItem.css
-        |        
+        |
         |--- index.js
 
+### Tips
 
-## Tips
-
-Create Components React assumes that you want all paths to a component and the component's name first character to be uppercase. If you input a component name or any paths that use lowercase chars, this package will convert it to the uppercase version. Examples: 
+Create Components React assumes that you want all paths to a component and the component's name first character to be uppercase. If you input a component name or any paths that use lowercase chars, this package will convert it to the uppercase version. Examples:
 
 `app` will turn into `App`
 
 `nav/navItem` will turn into `Nav/NavItem`
 
+## Settings
+
+create-components-react gives you the capability to make localized settings for each repo you wish to chose. These setting will change the behavior of how create-components-react functions. These settings would be like using the flags in the command line but it will be apply everytime a react component is made.
```

Comment:
These setting -> These settings
...but it will be apply... -> ...but it will be applied every time a react component is made

Ref: https://www.grammarly.com/blog/everytime-every-time/